api/envoy/config/route/v3/route_components.proto

```diff
@@ -513,6 +513,12 @@ message RouteMatch {
     // Note that CONNECT support is currently considered alpha in Envoy.
     // [#comment: TODO(htuch): Replace the above comment with an alpha tag.]
     ConnectMatcher connect_matcher = 12;
+
+    // If specified, the route is a path - separated prefix rule meaning that the
+    // *:path* header (without the query string) must either exactly match the
+    // path_separated_prefix or have it as a prefix, followed by '/'
```

Comment:
```suggestion
    // *path_separated_prefix* or have it as a prefix, followed by '/'
```

Reply:
please dont use `*` for inline literals - use double backticks ` `` `

same elsewhere in this PR where you have used `*` or `'`

Reply:
Will do, is there a guide for the formatting of comments?
I might look things over, to save reviewers some work.

Reply:
sort of - here https://github.com/envoyproxy/envoy/blob/main/api/CONTRIBUTING.md#documentation-guidelines

originally the guidance suggested using single backticks - and the codebase still has a lot of literals with either single backticks or asterisks

for reference here is the commit where i flipped the advice to use (the more semantically correct) backticks for literals

here https://github.com/envoyproxy/envoy/commit/eb18d71972e1f8af714d7843eb206108aad153c7